src/Sarif.UnitTests/SarifTransformerTests.cs

```diff
@@ -13,14 +13,15 @@ namespace Microsoft.CodeAnalysis.Sarif
 {
     public class SarifTransformerTests
     {
-        private static SarifLogVersionOne GetSarifLogVersionOne(string logText)
+        private static JsonSerializerSettings s_jsonSettings = new JsonSerializerSettings
```

Comment:
`readonly`

Reply:
Fixed

Reply:
I didn't mean to say it shouldn't be `static`, I meant it should also be `readonly` (`static readonly`).

Reply:
oh OK :)